test/tools/WebListener/Controllers/GetController.cs

```diff
@@ -29,8 +30,25 @@ public JsonResult Index()
                 {"args"   , args},
                 {"headers", headers},
                 {"origin" , Request.HttpContext.Connection.RemoteIpAddress.ToString()},
-                {"url"    , UriHelper.GetDisplayUrl(Request)}
+                {"url"    , UriHelper.GetDisplayUrl(Request)},
+                {"method" , Request.Method}
             };
+
+            if(Request.HasFormContentType){
```

Comment:
Typo - spaces skipped and the brace to next line.

Reply:
will fix

Reply:
fixed